en/contributing/dev_logging.md

```diff
@@ -0,0 +1,29 @@
+# Plugin/Test Logging
+
+DroneCore provides an API that developers can use for console logging in plugins and integration tests. 
+
+The API methods display a custom message, prepending a timestamp and the type of log message (e.g. debug) and appending the origin of the message (file and line number).
+
+> **Tip** The API should be considered "internal". It is not exported, or intended, for use in Dronecore apps (and we do not commit to maintaining compatibility in future versions).
```

Comment:
@julianoes Please in particular check that this is a true statement.

Reply:
Yes, I agree with this.